### PR TITLE
Works popup window

### PR DESCRIPTION
**Pull request by tonnymuchui**
Added JavaScript array to store all of the information for all projects. Implemented Js interaction, When the user clicks (or taps) the button to check project details, the popup with details about the project appears.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -81,132 +85,15 @@ <h1 class="content-text">I'm Tonny<br />Glad to see you!</h1>
...
-              </ul>
-              <button
-                class="btn"
-                onclick="window.location.href = 'https://github.com/tonnymuchui/Portfolio';"
-              >
-                See Project
-              </button>
-            </div>
-          </div>
+          <div class="container"></div>
```
Comment (ShoiraTa) [inline index.html:90]: - [ ] After the recent changes, I can see that the structure of project cards has been broken, please make sure that your layoput is same as provided on Figma:

Your design  | Figma design
------------- | -------------
Screenshot 2022-09-08 at 11 11 59 | Screenshot 2022-09-08 at 11 10 16

As you can see, project's text content should be on the left or on the right depending on the flow. But in your case, it is always on the right

File: js/script.js [hunk 2]
```diff
@@ -12,4 +13,185 @@ document.querySelectorAll('.nav-link').forEach((link) => link.addEventListener('
...
+  (card, index) => `
+<section class="works ${index}">
+<div class="work-card">
+<img
+  class="img"
+  src="${card.imageM}"
+  alt="SnapshootPortfolio02-img"
+/>
+<div class="align-center-text">
+  <h3 class="work-heading ${index}">${card.title}</h3>
```
Comment (ShoiraTa) [inline js/script.js:85]: - [ ] Please add a little more space below the project title and date info:

Your design  | Figma design
------------- | -------------
Screenshot 2022-09-08 at 11 15 43  | Screenshot 2022-09-08 at 11 15 26

**Review by ShoiraTa — COMMENTED** [review]
Hi team,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Modal window works as expected ✔️ 
- You passed all the Linters! ✔️ 
- Data is retrieved dynamically ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

- [ ] Please check the readme file, some sections look broken: 

Screenshot 2022-09-08 at 11 17 41


_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @tonnymuchui, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Correct use of GitHub flow. ✔️
 - Good modal design. ✔️ 

## Aspects to improve ♻️

- [ ] As the previous code reviewer mentioned, your README file looks broken. Please try to manipulate the images and the sections so it looks more professional. If you try to make the changes through Github, you can click on the 'Preview' button to see how it looks. Be aware of the attributes that you are applying to the images.
-Your README:
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — APPROVED** [review]
### Hi team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job fixing the README file 💯 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tonnymuchui** [comment]
I was requesting if it could be possible we spend like 10min on zoom you
help me to know how I will implement project card on the right

**Pull request merged**